[docs][development/getting-started]: Add a note about rebuilding the dev client (#19012)
expo/expo: docs/pages/development/getting-started.md

import { Tab, Tabs } from '~/components/plugins/Tabs';
import { Terminal } from '~/ui/components/Snippet';

> Development builds of your app are Debug builds of your project that includes the [`expo-dev-client`](https://www.npmjs.com/package/expo-dev-client) library. It allows you to develop and debug projects from Expo CLI or a compatible server.

Reviewer: 'includes' reads weird here. Is it the builds that include it, or the project?

Author: 🤔 I can see the wording being confusing. I think it will be more clear to say that Development builds use `expo-dev-client` since the library doesn't come included in the project and one has to manually install it.

> Development builds of your app are Debug builds of your project that includes the [`expo-dev-client`](https://www.npmjs.com/package/expo-dev-client) library. It allows you to develop and debug projects from Expo CLI or a compatible server.

Author: ```suggestion
Development builds of your app are Debug builds of your project. They use [`expo-dev-client`](https://www.npmjs.com/package/expo-dev-client) library, which allows you to develop and debug projects from Expo CLI or a compatible server.
```

Author: @kbrandwijk I changed the verbiage here.

</Tabs>

After creating the first build, [install it on your device](/build/internal-distribution.md).

> > **Note**: If you are using a library that contains native APIs, for example, [`expo-secure-store`](/versions/latest/sdk/securestore/), you will have to rebuild the development client. This is because the native code of the library is not included in the development client automatically when installing the library as a dependency to your project.

Reviewer: ```suggestion
> **Note**: If you add a library that contains native APIs to your project, for example, [`expo-secure-store`](/versions/latest/sdk/securestore/), you will have to rebuild the development client. This is because the native code of the library is not included in the development client automatically when installing the library as a dependency to your project.
```

<Terminal cmd={["$ npx expo start --dev-client"]} />

> Then, scan the resulting QR code with your system camera or QR code reader (if you want to develop it against a physical device). Alternatively, you can press <kbd>a</kbd> or <kbd>i</kbd> keys (to open the app in your Android or iPhone emulator or simulator).

Reviewer: What does 'it' refer to, the only subject in the sentence is the QR code.

```suggestion
Then, scan the resulting QR code with your system camera or QR code reader (if you want to develop against a physical device). Alternatively, you can press the <kbd>a</kbd> or <kbd>i</kbd> keys (to open the app in your Android emulator or iPhone simulator).
```

Author: the `it` is my mistake for overlooking the context.
